packages/fx-core/src/common/spec-parser/utils.ts

```diff
@@ -78,20 +77,19 @@ export function isSupportedSchema(schema: OpenAPIV3.SchemaObject): boolean {
     schema.type === "boolean" ||
     schema.type === "number"
   ) {
-    return true;
-  } else if (schema.type === "array") {
-    return false;
+    return schema.required ? 1 : 0;
   } else if (schema.type === "object") {
     const { properties } = schema;
+    let count = 0;
     for (const property in properties) {
-      const result = isSupportedSchema(properties[property] as OpenAPIV3.SchemaObject);
-      if (!result) {
-        return false;
-      }
+      const result = checkPostBodyRequiredParameters(
+        properties[property] as OpenAPIV3.SchemaObject
+      );
+      count += result;
```

Comment:
thought there might cause exception? the value can be NaN returned?

Reply:
NaN + any number would still NaN, so it would work

Reply:
do we validate such cases in validate() today?

Reply:
oh, realized we will filter out these APIs.